src/megameklab/com/util/EquipmentTableModel.java

```diff
@@ -341,7 +341,7 @@ public Object getValueAt(int row, int col) {
         } else if (col == COL_RANGE) {
             if (null != wtype) {
                 if (entity instanceof Aero) {
-                    switch (wtype.maxRange) {
+                    switch (wtype.getMaxRange(null)) {
```

Comment:
The null argument is for the mount, as is used for weapons like ATM and MML that vary the range depending on the type of ammo loaded. Passing null skips that check and just returns the calculated max range, which is identical to the previous behavior for heavy weapons, but now handles light and medium (e.g. infantry) weapons correctly.